cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/SortedReplicas.java

```diff
@@ -87,13 +90,13 @@
    *
    * @return the sorted replicas wrappers in the ascending order of their priority and score.
    */
-  NavigableSet<ReplicaWrapper> sortedReplicaWrappers() {
+  public NavigableSet<ReplicaWrapper> sortedReplicaWrappers() {
     ensureInitialize();
     return Collections.unmodifiableNavigableSet(_sortedReplicas);
   }
 
   /**
-   * Get a list of in the ascending order of their priority and score.
+   * Get the sorted replicas in the ascending order of their priority and score.
```

Comment:
Nit: If this applies to `master` can we move it to a separate patch for consistency?